### PR TITLE
Big PR 2fa, restructuring, status, erase all console.log, connection pages

### PR DESCRIPTION
**Pull request by emmanuel-lbs**
- 2fa -> we can set 2fa in profile page in onglet 2fa and if is set when we log we are redirect to 2fa log page (i dont have the variable 2faEnable:boolean when the user log , so 2fa is good i think but we can't test we have to wait update by Rozhou 
- restrucuturation of front/src/component/Profile for split the fille in 5 files is better like that 
- restructuration in front/srcs/styles -> clean the style ( new dir Pages/ new dir components/)
-  and new files index.scss si a include files for scss -> now no nedd to include scss in our .tsx files just include the new scss files in index.scss and its ok 
- the connection page have 2 button log/sign (but its the sme button in real) if user dont exist redirect to sign and if exist and  if 2faEnable redirect to page log and if 2faEnable == false redirect to /Home
- status update its better but not perfect update of status (Ingame not ok)
- i erase all console.log XD to many old log
Screenshot from 2023-02-14 11-01-46
Screenshot from 2023-02-14 11-03-34
Screenshot from 2023-02-14 11-04-16
Screenshot from 2023-02-14 11-04-41

### CODE REVIEW

File: back/src/app.gateway.ts [hunk 1]
```diff
@@ -21,4 +24,14 @@ export class AppGateway implements OnGatewayConnection, OnGatewayDisconnect {
...
 	}
+
+	@SubscribeMessage("status")
+	HandleStatus(@MessageBody() status: string, @ConnectedSocket() client: Socket)
+	{
+	  console.log("emit");
+	  this.userService.SetStatus(client.handshake.auth.user, status);
+	  console.log("emit");
+	  this.server.emit("ChangeStatus", {status, idChange: client.handshake.auth.user.id })
+	}
```
[the remark this answers is not in the document]
Comment (emmanuel-lbs) [reply back/src/app.gateway.ts:35]: pas vraiment mais apres peut etre

File: front/src/components/Header/people_list.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (Blodispic) [inline front/src/components/Header/people_list.tsx:1]: cool cool

File: front/src/components/Profile/History.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (Blodispic) [inline front/src/components/Profile/History.tsx:1]: Cool cool

**Review by Blodispic — APPROVED** [review]
Beaucoup de fichier avec des console.log en moins,
51 fichier modifié, va surement y'avoir des conflits, merci manu
Dans l'ensemble ca me parait ok.

**Comment by se-lee** [comment]
Wow it's a BIG PR ! Merci Manu!
(I could only check half of the changed files XD )
Just for your info, I still need some erased console.logs in chat.gateway so I might take those back when merging,
so please don't bother if there are still some logs on my next pull request

qr code looks very cooool
bravooooo

**Pull request merged**